packages/financial-templates-lib/src/price-feed/CreatePriceFeed.js

```diff
@@ -86,7 +86,9 @@ async function createPriceFeed(logger, web3, networker, getTime, config) {
       config.twapLength,
       config.lookback,
       getTime,
-      config.invertPrice // Not checked in config because this parameter just defaults to false.
+      config.invertPrice, // Not checked in config because this parameter just defaults to false.
+      config.poolDecimals,
```

Comment:
Is the pool decimals derivable by looking at the decimals of the two tokens in the pool? In what cases would this not work?

Reply:
You know I'm not sure how it actually works. Take the USDC/ETH pool for example which emits a `Sync` event with  `reserve0` in 6 decimal precision, and `reserve1` in 18 decimal precision. Perhaps the `poolDecimals` is equal to `reserve0`, unless the price is inverted?

Reply:
in the case of `USDC/ETH` is it 6 decimals? It could be the lower decimals of the two tokens within the pair.

Reply:
Its 6 decimals. I THINK its the lower of the two decimals, but I'm not sure if the ordering of `reserve0` or `reserve1` matters. This seems like I need to dig into it a bit more which is why I left it as a todo (same with the BalancerPriceFeed's poolPrecision FYI). Ideally I'd like this not to block this PR